docs/report_queries/invalid_entity_uri.md

```diff
@@ -0,0 +1,21 @@
+# Invalid Entity URI
+
+**Problem:** OBO entities are formatted http://purl.obolibrary.org/obo/IDSPACE_0000000. This format is assumed by many OBO tools. Often, accidentally typos cause entity to be ignored during processing.
```

Comment:
In other query docs, we've typically formatted the "problem" section as stating what exactly was done wrong. This is minor, but it would be nice to keep it consistent:

> An entity's URI is not formatted correctly ... etc. etc. etc.